content/survey_2022.ipynb

```diff
@@ -0,0 +1,4709 @@
+{
```

Comment:
<p>Should include the exact question we asked here, as in other sections.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/holoviz-dev/blog/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>